Make.versions

```diff
@@ -66,11 +66,11 @@ MAC_PACKAGE_VERSION=7.15.0.$(MAC_COMMIT_DISTANCE)
 # WARNING: Do **not** use versions higher than the available Xcode SDK or else we will have issues with mtouch (See https://github.com/xamarin/xamarin-macios/issues/7705)
 # When bumping the major macOS version in MACOS_NUGET_VERSION also update the macOS version where we execute on bots in jenkins/Jenkinsfile (in the 'node' element)
 
-IOS_NUGET_VERSION=14.5.100
-TVOS_NUGET_VERSION=14.5.100
-WATCHOS_NUGET_VERSION=7.4.100
-MACOS_NUGET_VERSION=11.3.100
-MACCATALYST_NUGET_VERSION=14.5.100
+IOS_NUGET_VERSION=15.0.100
+TVOS_NUGET_VERSION=15.0.100
+WATCHOS_NUGET_VERSION=8.0.100
+MACOS_NUGET_VERSION=12.0.100
+MACCATALYST_NUGET_VERSION=15.0.100
```

Comment:
For legacy
```
IOS_PACKAGE_VERSION=14.21.0.$(IOS_COMMIT_DISTANCE)
MAC_PACKAGE_VERSION=7.15.0.$(MAC_COMMIT_DISTANCE)
```
also needs to be updated

Reply:
Might as well include the maccore bump as the same time

Reply:
Fixed (both)